hedyorg/hedy
# 🪲 Insert cheatsheet code at the end

When selecting code from the cheat sheet, instead of simply deleting the user's existing code, now it is appended to the end of the user's code.

Fixes #4099

**How to test**

 * Go to Hedy's page and click the yellow button in one of the sections, and check that the code is appended to the end.
